packages/mui-base/src/Field/Validity/FieldValidity.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('<Field.Validity />', () => {
     fireEvent.change(input, { target: { value: 'test' } });
     fireEvent.blur(input);
 
-    const [data] = handleValidity.args[8];
+    const [data] = handleValidity.lastCall.args;
```

Comment:
@atomiks, could you please verify that with these changes the intent of these checks is the same?

Reply:
Yes, that's fine